src/components/Select.tsx

```diff
@@ -113,7 +113,6 @@ const Select = <T extends string>({
   return (
     <SelectWrapper
       role="button"
-      aria-label="Selected network"
```

Comment:
No more accessibility on this? :)

Reply:
Well, this was wrong to start with :P It's a generic `Select` component, not necessarily used for network selection.
The aria-label has to be set where the component is used (in `NetworkSwitch`). I'll do it :)